### PR TITLE
ios: verify connection security code

### PR DESCRIPTION
**Pull request by epoberezkin**

### CODE REVIEW

File: apps/ios/Shared/Views/Chat/VerifyCodeView.swift [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+                    Text(splitCode)
+                        .multilineTextAlignment(.leading)
+                        .font(.body.monospaced())
+                        .lineLimit(20)
+                        .padding(.bottom, 8)
+                }
+                .frame(maxWidth: .infinity, alignment: .center)
+
+                Text("To verify end-to-end encryption with your contact compare (or scan) the code on your devices.")
```
Comment (jr-simplex) [inline apps/ios/Shared/Views/Chat/VerifyCodeView.swift:52]: parameterize view with text so it says "group member" in place of "your contact" in groups?

Comment (jr-simplex) [reply apps/ios/Shared/Views/Chat/VerifyCodeView.swift:52]: full sentence not just subject

Comment (epoberezkin) [reply apps/ios/Shared/Views/Chat/VerifyCodeView.swift:52]: Don’t think it’s important

Comment (epoberezkin) [reply apps/ios/Shared/Views/Chat/VerifyCodeView.swift:52]: Nobody will verify group members I think…

File: apps/ios/Shared/Views/Chat/Group/GroupMemberInfoView.swift [hunk 2]
```diff
@@ -36,75 +38,92 @@ struct GroupMemberInfoView: View {
...
+        }
+    }
+
+    private func groupMemberInfoView() -> some View {
+        VStack {
+            List {
+                groupMemberInfoHeader(member)
+                    .listRowBackground(Color.clear)
 
+                Section {
```
Comment (jr-simplex) [inline apps/ios/Shared/Views/Chat/Group/GroupMemberInfoView.swift:54]: how does it look if there's no send and no verify code button?

Comment (jr-simplex) [reply apps/ios/Shared/Views/Chat/Group/GroupMemberInfoView.swift:54]: though the second one should always be present..

Comment (epoberezkin) [reply apps/ios/Shared/Views/Chat/Group/GroupMemberInfoView.swift:54]: As nothing

Comment (epoberezkin) [reply apps/ios/Shared/Views/Chat/Group/GroupMemberInfoView.swift:54]: Maybe a bit more white space. Verify is only present for active members, not while they are connecting.

Comment (epoberezkin) [reply apps/ios/Shared/Views/Chat/Group/GroupMemberInfoView.swift:54]: Btw why we are showing send message for inactive?

**Pull request merged**